### ISSUE TITLE
Anomalie

### ISSUE THREAD

**Issue by PhilippImhof** [huguesdevimeux/enigma-simulator#1]
La simulation (qui est très bien faite!) ne contient pas une anomalie de la vraie machine: le disque du milieu peut parfois avancer deux fois de suite. Voici un exemple avec les disques I, II et III:

ADU -> ADV -> AEW (le disque du milieu est entraîné par le disque à droite) 

Maintenant, le disque du milieu a atteint la position à laquelle il va entraîner le disque de gauche. En raison de l'anomalie (qui semble être due à la construction mécanique), ce mouvement surviendra lors de la prochaine lettre. Par conséquent, le disque du milieu tourne encore une fois et entraîne celui de gauche. Au lieu d'obtenir AEX, on aura donc BFX.

**Comment by huguesdevimeux** [comment]
Merci beaucoup pour ce commentaire (je viens de le voir, désolé pour ces deux semaines de retard) ! 

Je n’avais jamais entendu parlé de ça, c’est très intéressant. Mais est ce que cela était vrai pour les machines à plus de 3 rotors (je pense à celle de la Kriegsmarine, par exemple) ?

Je ne pense pas que je vais l’implémenter, mais en tout ça c’est toujours bon à savoir.

**Comment by PhilippImhof** [comment]
Oui, l'effet est présent dans chaque modèle de la machine, parce que c'est dû à la construction. Cependant, cela ne concerne que le deuxième disque (de droite) même pour la M4, car pour les machines à plus de trois rotors, les disques supplémentaires n'effectuent pas de rotation.

http://www.intelligenia.org/downloads/rotors1.pdf
